adobe_premiere_plugin/client/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <!--
+      Invert the code comments on the following two lines to use your local
+      version of Scrounger during development
+    -->
+    <meta http-equiv="refresh" content="0;url=https://scrounger.zorroa.com/" />
```

Comment:
Does the `refresh` + `content=0` combo instantly refresh the page on load?

Reply:
yeah, `refresh 0` is basically `redirect`

we'll have to remove that eventually though because Danny wants us to make this url dynamic to accommodate multiple hosted individual scrounger instances